### PR TITLE
CLOUD-51875 added network doctor

### PR DESCRIPTION
**Pull request by doktoric**
@akanto

### CODE REVIEW

File: include/deployer.bash [hunk 1]
```diff
@@ -201,6 +202,20 @@ doctor() {
     generate_uaa_check_diff verbose
 }
 
+network-doctor() {
+    ping 8.8.8.8 -c 1 &> /dev/null
```
Comment (lalyos) [inline include/deployer.bash]: please add a timeout for ping `-W 1`

File: include/deployer.bash [hunk 2]
```diff
@@ -201,6 +202,20 @@ doctor() {
...
+    if [[ "$?" -ne "0" ]]; then
+        error "Could not resolve 8.8.8.8."
```
Comment (lalyos) [inline include/deployer.bash]: probably "Couldnt reach" or "Couldnt ping" but "resolve is missleading" as there is no dns involved yet

File: include/deployer.bash [hunk 3]
```diff
@@ -201,6 +202,20 @@ doctor() {
...
+        _exit 1       
+    fi
+    {
+        curl -I https://github.com/sequenceiq/cloudbreak-deployer/releases/latest &> /dev/null
```
Comment (lalyos) [inline include/deployer.bash]: please add timeout to curl: `-m 1` and also the fail `-f` option

**Comment by doktoric** [comment]
@lalyos pls check now :)

**Comment by lalyos** [comment]
merged by hand

**Pull request closed without merging**